src/TestRunner-Core/RunExecutionMode.class.st

```diff
@@ -0,0 +1,34 @@
+"
+I ececute a test suite then return its TestResult.
+"
+Class {
+	#name : #RunExecutionMode,
+	#superclass : #AbstractExecutionMode,
+	#category : #'TestRunner-Core-Configuration'
+}
+
+{ #category : #execution }
+RunExecutionMode class >> executeCase: aTestCase [
+
+	| aResult |
+	aResult := TestResult new.
+	[ 
+	[ 
```

Comment:
It seems it is a full duplication of TestResult>>runCase: method. It can become: 
 ```Smalltalk
[aResult runCase: aTestCase] ensure: [ ... ]
```